src/hotspot/share/gc/shenandoah/shenandoahFreeSet.hpp

```diff
@@ -108,7 +144,7 @@ class ShenandoahFreeSet : public CHeapObj<mtGC> {
 
   void print_on(outputStream* out) const;
 
-  void reserve_regions(size_t to_reserve);
+  void reserve_regions(size_t young_reserve, size_t old_reserve);
```

Comment:
Documentation comment, preferably here, but if not then in the implementation as a block header.

Reply:
Added documentation.